src/ur5e_arm.cpp

```diff
@@ -1,5 +1,7 @@
 #include "ur5e_arm.hpp"
 
+#include <cmath>
```

Comment:
Would someday love to learn about best practices for imports in c.  I've just been shoving everything in the hpp and importing that but I'm sure thats frowned on